entrypoint.sh

```diff
@@ -1,11 +1,16 @@
 #!/bin/sh
 
+# set -xv
+set +H
+
 cd "$GITHUB_WORKSPACE" || exit
 export REVIEWDOG_GITHUB_API_TOKEN="${INPUT_GITHUB_TOKEN}"
 
+echo $INPUT_EXCLUDE;
```

Comment:
There is no reason to print `INPUT_EXCLUDE` unless debug mode or tracing is enabled.